doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -231,7 +229,29 @@ It defines the end-end Certification Process required for certifying the given V
 using this platform (providing above requirements), producing the test results to
 certification portal , review and granting the badge to this VNF.
 
-[More details](https://wiki.lfnetworking.org/display/LN/VNF+Certification+Lifecycle)
+The complete process is detailed below:
+
+1. *VNF Vendors* submit the VNF into OVP Lab for certification testing_guide
+
+2. Using VNF Test Platform, VNF will be tested with all required test cases Provided
+by *Test authors* by using the appropriate MANO and NFVi provided by Vendors. while

```

Comment:
Pls use NFVI consistently (instead of NFVi)

Reply:
Done.  Updated.  All references are now NFVI.  Thank you for reviewing.  Resolving/closing thread.